linux/memory_hot_add_basic/memory_hot_add_prepare.yml

```diff
@@ -1,35 +1,45 @@
 # Copyright 2021-2024 VMware, Inc.
 # SPDX-License-Identifier: BSD-2-Clause
 ---
-- include_tasks: ../../common/vm_get_mem_info.yml
+- name: "Get VM memory info"
+  include_tasks: ../../common/vm_get_mem_info.yml
 
-# Enable VM memory hotadd and set VM memory size to 2048 MB
-- block:
-    - include_tasks: ../utils/shutdown.yml
-    - include_tasks: ../../common/vm_enable_memory_hotadd.yml
+- name: "Enable VM memory hot add and set VM memory size to initial value"
+  when: >
+    (not vm_mem_hotadd_enabled) or
+    (vm_mem_size_mb | int != vm_initial_mem_mb | int)
+  block:
+    - name: "Shutdown guest OS"
+      include_tasks: ../utils/shutdown.yml
+    - name: "Enable VM memory hot add"
+      include_tasks: ../../common/vm_enable_memory_hotadd.yml
       when: not vm_mem_hotadd_enabled
 
-    - include_tasks: ../../common/vm_set_memory_size.yml
+    - name: "Set VM memory size to the initial value"
+      include_tasks: ../../common/vm_set_memory_size.yml
       vars:
         memory_mb: "{{ vm_initial_mem_mb }}"
       when: vm_mem_size_mb | int != vm_initial_mem_mb | int
 
-    - include_tasks: ../../common/vm_set_power_state.yml
+    - name: "Power on VM"
+      include_tasks: ../../common/vm_set_power_state.yml
       vars:
         vm_power_state_set: 'powered-on'
-    - include_tasks: ../../common/update_inventory.yml
-  when: >
-    (not vm_mem_hotadd_enabled) or
-    (vm_mem_size_mb | int != vm_initial_mem_mb | int)
+    - name: "Refresh in memory host IP"
```

Comment:
The `host` used here might be confusing. What about `"Update VM's guest IP in inventory"`